tests/test_json_io.py

```diff
@@ -200,13 +200,7 @@ def setUpClass(cls):
             ],
             "Event": ["timestamp", "event_type", "precedence"],
             "PluginEvent": ["timestamp", "event_type", "precedence"],
-            "UnplugEvent": [
-                "timestamp",
-                "event_type",
-                "precedence",
-                "station_id",
-                "session_id",
-            ],
+            "UnplugEvent": ["timestamp", "event_type", "precedence"],
```

Comment:
We need to check the EV associated with the UnplugEvent was serialized/deserialized.

Reply:
Where does this happen for PluginEvent? I just tried to copy the logic from it. Honestly with the new EVEvent class should we do that test on the base EVEvent class instead?